orttraining/orttraining/core/graph/adasum_optimizer_graph_builder.h

```diff
@@ -24,7 +24,7 @@ class AdasumOptimizerGraphBuilder : public AllreduceOptimizerGraphBuilder {
       GraphAugmenter::GraphDefs& graph_defs,
       std::vector<ArgDef>& weight_argdefs,
       std::vector<ArgDef>& gradient_argdefs,
-      std::unordered_set<std::string>& optimizer_state_initializer_names,
+      std::unordered_map<std::string, std::vector<std::string>>& weight_to_opt_mapping,
```

Comment:
should this be a ``` std::unordered_map<std::string, std::unordered_set<std::string>>``` instead?

Reply:
This function only needs string. tensor is passed in another parameter